route.go

```diff
@@ -31,7 +31,7 @@ func NewRoute(config config.Config, parameters map[string]any) (*Route, error) {
 	gin.DisableBindValidation()
 	engine := gin.New()
 	engine.MaxMultipartMemory = int64(config.GetInt("http.drivers.gin.body_limit", 4096)) << 10
-	engine.Use(gin.Recovery()) // recovery middleware
+	// engine.Use(gin.Recovery()) // recovery middleware
```

Comment:
We can keep this line, it will be used when the user doesn't set the global recover and the timeout middleware.